### PR TITLE
Prepare v1.3.14

### PR DESCRIPTION
**Pull request by line-o**

### CODE REVIEW

File: repo.xml [hunk 1]
```diff
@@ -14,6 +14,11 @@
     <permissions user="packageservice" password="packageservice" group="packageservice" mode="rwxrwxr-x"/>
 
     <changelog>
+        <change version="1.3.14">
+            <ul xmlns="http://www.w3.org/1999/xhtml">
+                <li>Load packages over a secure connection by default<a href="https://github.com/eXist-db/existdb-packageservice/pull/24">#24</a></li>
```
Comment (joewiz) [inline repo.xml:19]: I think ` - ` is needed between the description and the link

Comment (line-o) [reply repo.xml:19]: OK, I'll add it

**Comment by line-o** [comment]
@joewiz done!

**Review by joewiz — APPROVED** [review]

**Comment by joewiz** [comment]
@line-o Thanks!

**Pull request merged**